frontend/src/container/CreateAlertChannels/index.tsx

```diff
@@ -457,29 +470,41 @@ function CreateAlertChannels({
 							message: 'Error',
 							description: t('test_unsupported'),
 						});
+						testSuccess = false;
 						setTestingState(false);
 						return;
 				}
 
 				if (response && response.statusCode === 200) {
+					testSuccess = true;
 					notifications.success({
 						message: 'Success',
 						description: t('channel_test_done'),
 					});
 				} else {
+					testSuccess = false;
 					notifications.error({
 						message: 'Error',
 						description: t('channel_test_failed'),
 					});
 				}
 			} catch (error) {
+				testSuccess = false;
```

Comment:
do we need this variable? can we not get the status from the response itself ?

Reply:
yes, majorly because of 2 reasons - 
- response is present in inner block scope
- success will captured by response status, but error are coming from different areas, like default case of switch etc.

Reply:
Ignored the try-catch as we dont need to track event on that, and now used response status directly and removed variable